tsdfileapi/api.py

```diff
@@ -119,6 +119,10 @@ def set_config() -> None:
     define('rabbitmq_cache', queue.Queue())
     define('maintenance_mode_enabled', False)
     define('request_log', _config.get('request_log'))
+    if 'backlog' in _config:
+        tornado.netutil.DEFAULT_BACKLOG = _config['backlog']
```

Comment:
could be more elegant to simply have:
```python
tornado.netutil.DEFAULT_BACKLOG = _config.get('backlog')
tornado.netutil.ACCEPT_CALLS_PER_EVENT_LOOP = _config.get('accept_calls_per_event_loop')
```
since if those are not in the config, they will just be `None`

Reply:
It wouldn't be having exactly the same effect -- if Tornado (upstream) decides to swap their defaults to something else than `None` (in a future version, assuming they even merge our changes into upstream), we'd be overriding their values even if our configuration file doesn't specify any values (`get` returns `None`). That's why I wrote the statements the way I did.

Reply:
Fair enough, but to be exact we depend on a fork, not upstream.

Reply:
Come to think of it, the `if 'backlog' in _config` is actually buggy, since `_config` is a dict. It should be either `if 'backlog' in _config.keys()` or `if _config.get('backlog')`.

Reply:
I believe `if 'backlog' in _config` and `if 'backlog' in _config.keys()` result in the same check being ran in Python, but the latter `if _config.get('backlog')` might be better if you want to assure an actual value is assigned for this optional setting.

Reply:
I see your point. I was just assuming that upstream will merge our changes and we won't be staying on forked 6.0.3 for too long; if they don't [merge our changes], well, then your point comes clearly across -- we then are either forced to stay on our fork or go back to using upstream Tornado and back to start of figuring out how to control backlog values.

Reply:
Oh these are all nitpicks anyways, so no worries

Reply:
These commits, by the nature of their content, may end up being more ephemeral than most things :)

Reply:
`'backlog' in _config` is indeed equivalent to `'backlog' in _config.keys()`: https://docs.python.org/3/reference/datamodel.html#object.__contains__ (emphasis on "For mapping objects, this should consider the keys of the mapping rather than the values or the key-item pairs.")